src/Core/Domain/Currency/Exception/CurrencyConstraintException.php

```diff
@@ -72,10 +72,21 @@ class CurrencyConstraintException extends CurrencyException
      */
     const EMPTY_SYMBOL = 7;
 
+    /**
+     * Code used when trying to insert a currency referenced in CLDR database but ISO code and
+     * numeric ISO code are not matching the expected ones
+     */
+    const ISO_CODES_MISMATCH = 8;
+
     /**
      * Code used when an invalid precision is used (positive integer expected)
      *
      * @see Precision
      */
-    const INVALID_PRECISION = 8;
+    const INVALID_PRECISION = 9;
```

Comment:
@PululuK Please don't change value for constant.

Reply:
See https://github.com/jolelievre/PrestaShop/blob/currency-edition/src/Core/Domain/Currency/Exception/CurrencyConstraintException.php#L86 
Merged here : https://github.com/PrestaShop/PrestaShop/pull/14972

Reply:
:ok_hand:

Reply:
By cons it is very strange, because I do not have that on the develop branch https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Domain/Currency/Exception/CurrencyConstraintException.php#L80, while it  https://github.com/jolelievre/PrestaShop/blob/currency-edition/src/Core/Domain/Currency/Exception/CurrencyConstraintException.php#L86 was merged  here #14972

ping @jolelievre @PierreRambaud

Reply:
Correct, I checked ;) Removed here : https://github.com/PrestaShop/PrestaShop/commit/ca475439c846940fe9e42254bf0dd04204a11a6e#diff-f5c3c83f1af68a05ac6ab2068e1b9c6d

Could you adapt your code ?

Reply:
Done !